src/vmm/src/vstate/vcpu/aarch64.rs

```diff
@@ -233,7 +233,7 @@ impl KvmVcpu {
 }
 
 /// Structure holding VCPU kvm state.
-#[derive(Clone, Default, Versionize)]
+#[derive(Debug, Default, Clone, Versionize)]
```

Comment:
nit: the second commit `refactor: simplified code for saving aarch64 regs` looks like a fixup to the previous one.

Reply:
fixed, have no idea why there were two of them